Description: tokio related code uses unstable feature `std::task::ready!`
Last-Update: 2023-02-10
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -46,12 +46,6 @@
     "docsrs",
 ]
 
-[dependencies.tokio]
-version = "1.21"
-features = ["fs"]
-optional = true
-default_features = false
-
 [dev-dependencies.serde_json]
 version = "1.0.64"
 
